WordPress/Classes/Services/BlogService.m

```diff
@@ -831,7 +830,15 @@ - (void)updateMultiAuthor:(BOOL)isMultiAuthor forBlog:(NSManagedObjectID *)blogO
         if (!blog) {
             return;
         }
-        blog.isMultiAuthor = isMultiAuthor;
+        blog.isMultiAuthor = users.count > 1;
+        if (blog.username != nil) {
```

Comment:
This did not work for me, debugging I found that the issue was that my blogs did not have the username field set up,  so this code never executed (effectively removing author filtering for .com blogs because `blog.userID` is never set).
I think the problem is that `blog.username` is only set up in `- (void)syncBlogWithUsername:(NSString *)username
                    password:(NSString *)password
                      xmlrpc:(NSString *)xmlrpc
                     options:(NSDictionary *)options
                finishedSync:(void(^)())finishedSync` so we can not expect it to be properly populated for .com or JP sites.